core/templates/pages/subtopic-viewer-page/navbar-breadcrumb/subtopic-viewer-navbar-breadcrumb.component.html

```diff
@@ -24,4 +24,14 @@
     display: flex;
     flex-direction: column;
   }
+
+  .oppia-subtopic-title-text {
```

Comment:
As I mentioned before, please make sure that all classes, here and below, are prefixed correctly. This occurs in other files too. Please fix and, when replying to this comment, confirm that you have fixed all the cases.